packages/help-center/src/hooks/use-get-history-chats.ts

```diff
@@ -19,6 +24,11 @@ interface UseGetHistoryChatsResult {
 	archivedConversations: Conversations;
 }
 
+const getLastMessageReceived = ( conversation: OdieConversation | ZendeskConversation ) => {
+	const lastMessage = getLastMessage( { conversation } );
+	return ( lastMessage?.received || 0 ) * 1000;
+};
+
```

Comment:
I would be tempted to document the format of the timestamp, especially since our data structure is not consistent:

```suggestion
/**
 * Retrieves the date the last message from the specified conversation was received.
 *
 * @returns The timestamp in milliseconds (e.g. 1745936539027), or 0 if not available
 */
const getLastMessageReceived = ( conversation: OdieConversation | ZendeskConversation ) => {
	const lastMessage = getLastMessage( { conversation } );
	return ( lastMessage?.received || 0 ) * 1000;
};

```